components/Card.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="card" v-bind:style="{
```

Comment:
Tu veux pas m'écouter toi
```suggestion
  <div class="card" :style="{
```

Reply:
Oupsi je l'avais laissé trainé